.github/workflows/standard_R_ci.yaml

```diff
@@ -0,0 +1,110 @@
+on:
+  push:
+    branches:
+      - master
+      - continuous_integration
+  pull_request:
+    branches:
+      - master
+      - continuous_integration
+
+name: standard_ci
+
+jobs:
+  standard_ci:
+    runs-on: ${{ matrix.config.os }}
+
+    name: ${{ matrix.config.os }} (${{ matrix.config.r }})
+
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - {os: macOS-latest,   r: 'devel'}
+          - {os: macOS-latest,   r: 'release'}
+          - {os: windows-latest, r: 'release'}
+          - {os: ubuntu-16.04,   r: '4.0', rspm: "https://packagemanager.rstudio.com/cran/__linux__/xenial/latest"}
+          - {os: ubuntu-16.04,   r: '3.6', rspm: "https://packagemanager.rstudio.com/cran/__linux__/xenial/latest"}
+        
+    env:
+      R_REMOTES_NO_ERRORS_FROM_WARNINGS: true
+      RSPM: ${{ matrix.config.rspm }}
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install GTK for Linux
+        if: runner.os == 'Linux'
+        run: |
+          sudo apt-get install gtk2.0
+      
+      - name: Install GTK for Mac OSX
+        if: runner.os == 'macOS'
+        run: |
+          brew install gtk+
+        
+      - name: Install GTK for Windows
+        if: runner.os == 'Windows'
+        run: |
+          choco install gtk-runtime
+
```

Comment:
GTK might only be needed for playwith, in which case we'll remove this later?
Or did you see it was needed for something else?

Reply:
Yes, I believe so. We can remove this after the `playwith` dependency has been sorted out.

Reply:
Done if you'd like to remove this prior to merging this PR.